packages/pieces/sendy/src/lib/actions/subscribe.ts

```diff
@@ -0,0 +1,73 @@
+import { createAction, Property, Validators } from "@activepieces/pieces-framework";
+import { subscribe } from "../api";
+import { buildListDropdown } from "../props";
+import { sendyAuth, SendyAuthType } from "../auth";
+
+export const subscribeAction = createAction({
+	name        : 'subscribe',
+	auth        : sendyAuth,
+	displayName : 'Subscribe',
```

Comment:
Can you please rename to `Add Subscriber`

Reply:
I don't agree with this. The subscriber may already exist but is just subscribing to a new list.